### ISSUE TITLE
Importer not reporting syntax errors

### ISSUE THREAD

**Issue by cmoesel** [FHIR/sushi#85]
Sometimes the importer ignores syntax errors and doesn't report them to the user.

For example:
```
ValueSet: 				YesNoUnknownVS
Description:			"A value set containing yes, no, and unknown."
SCT#373066001			"Yes"
SCT#373067005			"No"
SCT#261665006			"Unknown"
```

This is missing the `*` before each line that adds a code.  The importer should report this, but instead just skips those lines so you end up with a valueset having no codes in it.

**Comment by markkramerus** [comment]
It is actually more insidious than described by Chris. If there is a rule without an asterisk, processing on the file STOPS completely. The rule is not ignored, it represents the death rattle of the parser.

**Comment by cmoesel** [comment]
Fixed in 0.6.1
